web/src/beta/features/Editor/Visualizer/convert-story.ts

```diff
@@ -41,6 +41,7 @@ export const convertStory = (story?: GqlStory): Story | undefined => {
     id: story.id,
     title: story.title,
     position: story.panelPosition === "RIGHT" ? "right" : "left",
+    bgColor: story.bgColor ?? "#f1f1f1",
```

Comment:
You also have the fallback `#f1f1f1` being done in the CSS. Just set bgColor here or in the CSS, not both. (preferably take a look at padding/gap/other fallbacks and see where we set them and do bgColor there too 🙏🏻

Reply:
done, I will keep it here